src/couch/src/couch_server.erl

```diff
@@ -230,7 +235,13 @@ init([]) ->
     ok = config:listen_for_changes(?MODULE, nil),
     ok = couch_file:init_delete_dir(RootDir),
     hash_admin_passwords(),
-    ets:new(couch_dbs, [set, protected, named_table, {keypos, #entry.name}]),
+    ets:new(couch_dbs, [
+        set,
+        protected,
+        named_table,
+        {keypos, #entry.name},
+        {read_concurrency, true}
```

Comment:
Wonder if this will slow down updates to the table. I forgot what the trade-offs for read_concurrency are...

Reply:
If memory serves, its the flip between read and write requests which is expensive which we already do exhaustively so I don't think it'd be that big of a deal.